pydov/types/interpretaties.py

```diff
@@ -365,3 +365,168 @@ def from_wfs_element(cls, feature, namespace):
             )
 
         return hydstrat
+
+
+class LithologischeBeschrijvingLaag(AbstractDovSubType):
+
+    _name = 'lithologische_beschrijving_laag'
+    _rootpath = './/lithologischebeschrijving/laag'
+
+    _fields = [{
+        'name': 'diepte_laag_van',
+        'source': 'xml',
+        'sourcefield': '/van',
+        'definition': 'Diepte van de bovenkant van de laag lithologische '
+                      'beschrijving in meter.',
+        'type': 'float',
+        'notnull': False
+    }, {
+        'name': 'diepte_laag_tot',
+        'source': 'xml',
+        'sourcefield': '/tot',
+        'definition': 'Diepte van de onderkant van de laag lithologische '
+                      'beschrijving in meter.',
+        'type': 'float',
+        'notnull': False
+    }, {
+        'name': 'beschrijving',
+        'source': 'xml',
+        'sourcefield': '/beschrijving',
+        'definition': 'Lithologische beschrijving van de laag in vrije tekst '
+                      '(onbeperkt in lengte)',
+        'type': 'string',
+        'notnull': False
+    }]
+
+    def __init__(self):
+        """Initialisation."""
+        super(LithologischeBeschrijvingLaag, self).__init__(
+            'lithologische_beschrijving_laag')
+
+    @classmethod
+    def from_xml_element(cls, element):
+        """Build an instance of this subtype from a single XML element.
+
+        Parameters
+        ----------
+        element : etree.Element
+            XML element representing a single record of this subtype.
+
+        """
+        laag = LithologischeBeschrijvingLaag()
+
+        for field in cls.get_fields().values():
+            laag.data[field['name']] = laag._parse(
+                func=element.findtext,
+                xpath=field['sourcefield'],
+                namespace=None,
+                returntype=field.get('type', None)
+            )
+
+        return laag
+
+
+class LithologischeBeschrijvingen(AbstractDovType):
+    """Class representing the DOV data type for 'lithologische
+    beschrijvingen' interpretations."""
+
+    _subtypes = [LithologischeBeschrijvingLaag]
+
+    _fields = [{
+        'name': 'pkey_interpretatie',
+        'source': 'wfs',
+        'sourcefield': 'Interpretatiefiche',
+        'type': 'string'
+    }, {
+        'name': 'pkey_boring',
+        'source': 'custom',
```

Comment:
This can be changed to `wfs`, since we don't need the logic to split `Proeffiche` into either `pkey_boring` or `pkey_sondering` anymore (since we can assume all will be `pkey_boring`).